app/src/main/java/org/gdg/frisbee/android/utils/Utils.java

```diff
@@ -218,7 +219,12 @@ public static boolean canLaunch(Context context, final Intent viewUrlIntent) {
 
     @SuppressWarnings("deprecation")
     public static Intent createExternalIntent(Context context, Uri uri) {
-        Intent intent = new Intent(Intent.ACTION_VIEW, uri);
+        CustomTabsIntent customTabsIntent = new CustomTabsIntent.Builder()
+                .setToolbarColor(context.getResources().getColor(R.color.theme_primary))
+                .setShowTitle(true)
+                .build();
+        Intent intent = customTabsIntent.intent;
+        intent.setData(uri);

```

Comment:
No, launchUrl is just a convenience method
On 5 Sep 2015 00:10, "Said Tahsin Dane" <notifications@github.com> wrote:

> In app/src/main/java/org/gdg/frisbee/android/utils/Utils.java
> <https://github.com/gdg-x/frisbee/pull/482#discussion_r38798489>:
>
> > @@ -218,7 +219,12 @@ public static boolean canLaunch(Context context, final Intent viewUrlIntent) {
> >
> >      @SuppressWarnings("deprecation")
> >      public static Intent createExternalIntent(Context context, Uri uri) {
> > -        Intent intent = new Intent(Intent.ACTION_VIEW, uri);
> > +        CustomTabsIntent customTabsIntent = new CustomTabsIntent.Builder()
> > +                .setToolbarColor(context.getResources().getColor(R.color.theme_primary))
> > +                .setShowTitle(true)
> > +                .build();
> > +        Intent intent = customTabsIntent.intent;
> > +        intent.setData(uri);
>
> Don't we have to call launchUrl function of customTabs to make it work?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gdg-x/frisbee/pull/482/files#r38798489>.
>